docker/docker-compose.yml

```diff
@@ -0,0 +1,143 @@
+version: '3.2'
```

Comment:
Is it normal to have this new docker-compose.yaml here? I though this PR was only about reducing the image sizes?

Reply:
No, I think I messed up in the confilct resolution when merging master. Should not be there.

Reply:
@rth 
Hmm, since I had pushed the merge button on the PR a second time, there's another merge that piled up on top, and I personally do not feel confident enough to rewrite the history there. ok with you if I delete the unwanted file and push the commit or do you prefer to so a rewrite (all the commits will be squashed in the end).

Reply:
No of course, you can just remove the file and push another commit. We'll squash when merging by default.